core/utils.gd

```diff
@@ -0,0 +1,19 @@
+extends Object
```

Comment:
No need to extend from object, and could cause memory leak if someone, somehow, want to spawn this script for no reason.

Reply:
What do I extend from?